rotkehlchen/api/rest.py

```diff
@@ -1326,10 +1326,13 @@ def ping() -> Response:
     @require_loggedin_user()
     def import_data(
             self,
-            source: Literal['cointracking.info'],  # pylint: disable=unused-argument
+            source: str,
```

Comment:
Why change the type here? Just add one more literal.

e.g.: 

`source: Literal['cointracking.info', 'crypto.com'],`

Reply:
I didn't know how that Literal type was working and the type test was failing so I used a more generic type trying to fix it. Thanks!